### PR TITLE
[ML-11407] Update sparkdl OSS repo to 2.0.0 and only keep the dummy HorovodRunner

### PR DESCRIPTION
**Pull request by WeichenXu123**
Update sparkdl OSS repo to 2.0.0 and only keep the dummy HorovodRunner

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -84,4 +84,3 @@ To apply the Apache License to your work, attach the following boilerplate notic
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
-
```
Comment (liangz1) [inline LICENSE:87]: This deletion seems not necessary.

File: sparkdl/horovod/__init__.py [hunk 2]
```diff
@@ -13,3 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+# pylint: disable=wrong-import-position
+# pylint: disable=invalid-name
```
Comment (liangz1) [inline sparkdl/horovod/__init__.py:17]: Why do we need these given there is no import or name here?

File: sparkdl/__init__.py [hunk 3]
```diff
@@ -12,3 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+# pylint: disable=wrong-import-position
+# pylint: disable=invalid-name
```
Comment (liangz1) [inline sparkdl/__init__.py:17]: Why do we need these given there is no import here? What is the invalid name?

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        "Environment :: Console",
+        "License :: OSI Approved :: BSD License",
+        "Operating System :: Unix",
+        "Programming Language :: Python :: 3",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Software Development",
+    ],
+    platforms=["Linux"],
+    license="BSD",
+    keywords="spark deep learning horovod model distribbuted training",
```
Comment (liangz1) [inline setup.py:40]: typo: distribbuted

File: run-tests.sh [hunk 5]
```diff
@@ -101,9 +61,8 @@ for noseOptions in $noseOptionsArr
 do
   echo "============= Running the tests in: $noseOptions ============="
   # The grep below is a horrible hack for spark 1.x: we manually remove some log lines to stay below the 4MB log limit on Travis.
```
Comment (liangz1) [inline run-tests.sh:63]: Do we still support spark 1.x?

File: run-tests.sh [hunk 6]
```diff
@@ -101,9 +61,8 @@ for noseOptions in $noseOptionsArr
...
-  $PYSPARK_DRIVER_PYTHON \
+  python \
```
Comment (liangz1) [inline run-tests.sh:64]: Do we need to enforce `python3` here?

File: setup.py [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+setuptools.setup(
+    name='sparkdl',
+    version=sparkdl.__version__,
+    packages=setuptools.find_packages(exclude=exclude_packages),
+    url="https://github.com/databricks/spark-deep-learning",
+    author="Weichen Xu",
+    author_email="weichen.xu@databricks.com",
+    description="Deep Learning Pipelines for Apache Spark",
+    long_description="",
```
Comment (liangz1) [inline setup.py:26]: Do we want to mention the fact that this package is only for testing purposes?

File: sparkdl/horovod/__init__.py (hunk 2 above)
Comment (WeichenXu123) [reply sparkdl/horovod/__init__.py:17]: It is minor one, we can leave it to reduce conflicts.

File: sparkdl/__init__.py (hunk 3 above)
Comment (WeichenXu123) [reply sparkdl/__init__.py:17]: ditto

File: LICENSE (hunk 1 above)
Comment (WeichenXu123) [reply LICENSE:87]: ditto

**Pull request merged**